backup/statistics.go

```diff
@@ -69,14 +69,16 @@ func GenerateAttributeStatisticsQueries(table Table, attStat AttributeStatistic)
 		attributeSlotsQueryStr = generateAttributeSlotsQuery4(attStat)
 	}
 
-	attributeQueries = append(attributeQueries, fmt.Sprintf(`DELETE FROM pg_statistic WHERE starelid = %s AND staattnum = %d;`, starelidStr, attStat.AttNumber))
-	attributeQueries = append(attributeQueries, fmt.Sprintf(`INSERT INTO pg_statistic VALUES (
+	attributeQueries = append(attributeQueries, fmt.Sprintf(`DELETE FROM pg_statistic
+WHERE starelid = %s AND staattnum = (SELECT attnum FROM pg_attribute WHERE attrelid = %s AND attname = '%s');`, starelidStr, starelidStr, utils.EscapeSingleQuotes(attStat.AttName)))
```

Comment:
Try rewriting your query to avoid double use of starelidStr, for example
```sql
DELETE FROM pg_statistic WHERE starelid, staattnum IN (
	SELECT attrelid, attnum FROM pg_attribute WHERE attrelid = %s AND attname = '%s'
)
```